realsense_ros_camera/src/base_realsense_node.cpp

```diff
@@ -964,15 +1119,15 @@ void BaseRealSenseNode::publishStaticTransforms()
 
         float3 trans{ex.translation[0], ex.translation[1], ex.translation[2]};
         quaternion q1{Q.x(), Q.y(), Q.z(), Q.w()};
-        publish_static_tf(transform_ts_, trans, q1, _base_frame_id, _frame_id[COLOR]);
+        publish_static_tf(transform_ts_, trans, q1, _frame_id[DEPTH], _frame_id[COLOR]);
```

Comment:
Why are we doing transforms from depth to other, instead of base_frame_id

Reply:
Because we have our own camera_link, which is a physical location of the camera (left bottom corner for R200 and screw mounting point for D435), and thus we provide the tf from camera_link to the first reference link the driver provides (depth frame in this case).